packages/@azure/servicebus/data-plane/test/batchReceiver.spec.ts

```diff
@@ -307,10 +319,22 @@ describe("Batch Receiver - Complete/Abandon/Defer/Deadletter normal message", fu
 
     const deadLetterMsgs = await deadLetterClient.getReceiver().receiveBatch(1);
 
-    should.equal(Array.isArray(deadLetterMsgs), true);
-    should.equal(deadLetterMsgs.length, 1);
-    should.equal(deadLetterMsgs[0].body, testMessages.body);
-    should.equal(deadLetterMsgs[0].messageId, testMessages.messageId);
+    should.equal(
+      Array.isArray(deadLetterMsgs),
+      true,
+      "`ReceivedMessages` from Deadletter is not an array"
```

Comment:
This should be `DeadLetterMessages` not `ReceivedMessages`

On second thoughts, I was wondering if we should ditch the check for `Array.isArray`
If the received object is not an array of messages, then further down when the check for individual messages and test the body/messageId will anyway fail :)

Reply:
> ReceivedMessages from Deadletter is not an array

 ->
> Messages received from Deadletter is not an array.
or
Received DeadLetterMessages is not an array.

Which one seems better ?

To remove or not to remove `Array.isArray` check ?

Reply:
I would still say we dont need the Array check.
But at the same time, its not worth spending too much time either :) I am ok with not making any changes here

Reply:
Yeah, since the main reason why we need this PR is because the test results don't trace back to the line number, I'm okay with what we have so far. 
I would open another issue to improve the describe and it messages.